upgrade/scripts/upgrade/prepare-assets.sh

```diff
@@ -143,8 +143,10 @@ if [[ $state_recorded == "0" ]]; then
         rm -rf "${TARBALL_FILE}"
     fi
 
-    #copy the canu RPM into the sle-15sp3 directory
-    cp /etc/cray/upgrade/csm/${CSM_REL_NAME}/tarball/${CSM_REL_NAME}/rpm/cray/csm/sle-15sp4/x86_64/canu-*.rpm /etc/cray/upgrade/csm/${CSM_REL_NAME}/tarball/${CSM_REL_NAME}/rpm/cray/csm/sle-15sp3/x86_64
+    #copy the canu RPM into the sle-15sp3 directory, but only for CSM v1.3.2
+    if [[ $CSM_REL_NAME == "csm-1.3.2" ]]; then
```

Comment:
```suggestion
    if [[ "$CSM_RELEASE" =~ *1.3.2* ]]; then
```

Use a regex on the `CSM_RELEASE` value, since it's just the literal version string. The regex will help catch all of our alphas/betas/rcs (not that anyone should be using those).

```bash
ncn-m001:~ # CSM_RELEASE=1.3.2
ncn-m001:~ # if [[ $CSM_RELEASE =~ .*1.3.2.* ]]; then
> echo yes
> fi
yes
ncn-m001:~ # CSM_RELEASE=1.3.2-alpha.2
ncn-m001:~ # if [[ $CSM_RELEASE =~ .*1.3.2.* ]]; then echo yes; fi
yes
ncn-m001:~ # CSM_RELEASE=1.3.3
ncn-m001:~ # if [[ $CSM_RELEASE =~ .*1.3.2.* ]]; then echo yes; fi
ncn-m001:~ #
```